CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## Important Notes
 
+- [#911](https://github.com/oauth2-proxy/oauth2-rpoxy/pull/911) Validate provider type on startup.
```

Comment:
Can we move this to the change since v6.1.1 section please? Having it in breaking is good enough additional highlighting, but all changes should be in changes since v6.1.1

Reply:
Sure :)

Reply:
Fixed.